anyway/parsers/streets.py

```diff
@@ -0,0 +1,90 @@
+import requests
+import json
+from typing import Iterable, Dict, Any, List
+from anyway.models import Streets
+from anyway.app_and_db import db
+import logging
+
+CBS_STREETS_RESOURCES_URL = "https://data.gov.il/api/3/action/package_show?id=321"
+RESOURCE_NAME = "רשימת רחובות בישראל - מתעדכן"
+BASE_GET_DATA_GOV = "https://data.gov.il/dataset/321"
+RESOURCE_DOWNLOAD_TEMPLATE = (
+    "https://data.gov.il/api/3/action/datastore_search?resource_id={id}&limit=100000"
```

Comment:
@ziv17 this limit is too small (and is it necessary to add a limit? perhaps we can just query all data?) since today there are 61455 records in the table, and I assume we can reach a time, at some point where 100000 is not enough.

Reply:
The default limit is 32k, so we must supply a limit. I did not find a way to work with pagination. I will set the limit to 1M.